apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -40,6 +40,7 @@ const {
 } = require('@open-condo/keystone/test.utils')
 const { makeClient } = require('@open-condo/keystone/test.utils')
 const { WRONG_TEXT_FORMAT, UNEQUAL_CONTEXT_ERROR } = require('@condo/domains/common/constants/errors')
+const { services } = require('@condo/domains/billing/utils/testData/servicesData')
```

Comment:
Do we have anything like this in other domains? 

I'd move it simly to testutils